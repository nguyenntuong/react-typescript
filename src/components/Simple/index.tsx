import { composeInferTypeMemo } from 'utils/composeInferType';

export interface ISimpleProps {}
function Simple({}: ISimpleProps): React.FunctionComponentElement<ISimpleProps> {
    return <div>Simple</div>;
}
export default composeInferTypeMemo(Simple)();
