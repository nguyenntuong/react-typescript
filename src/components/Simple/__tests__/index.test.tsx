import { shallow } from 'enzyme';
import Simple from '..';
describe('<Simple />', () => {
    it('Render match snapshot', () => {
        const com = shallow(<Simple />);
        expect(com.html()).toMatchSnapshot();
    });
});
