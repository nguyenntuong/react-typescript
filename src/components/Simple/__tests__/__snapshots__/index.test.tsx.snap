// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Simple /> Render match snapshot 1`] = `"<div>Simple</div>"`;
